#File generated by Pentaho Transator for package 'org.pentaho.di.trans.steps.addsequence' in locale 'es_AR'
#
#
#Fri Oct 26 18:43:28 GMT-03:00 2007
AddSequenceDialog.NoValidConnectionError.DialogMessage=\u00A1Seleccione una conexi\u00F3n v\u00E1lida\!
AddSequenceDialog.Log.GettingKeyInfo=obteniendo informaci\u00F3n de la clave...
AddSequence.Log.WriteRow=Se escribi\u00F3 la fila \#
AddSequence.Log.ConnectedDB=Conectado a la base de datos...
AddSequence.Exception.CouldNotFindNextValueForSequence=No se ha podido encontrar el siguiente valor de la secuencia \: 
AddSequence.Log.ErrorInStep=Este paso no puede continuar debido a un error\: 
AddSequenceDialog.TargetSchema.Label=Nombre de esquema
AddSequenceDialog.UseDatabase.Label=\u00BFUtilizar base datos para obtener secuencia? 
AddSequenceMeta.ErrorMessage.UnableToConnectDB=No se ha podido conectar a la base de datos para verificar el estado de la tabla.
AddSequenceDialog.UseCounter.Tooltip=Seleccione si quiere utilizar un contador interno de Kettle para calcular el valor de la secuencia.
AddSequence.Log.CouldNotConnectToDB=Este paso no se ha podido conectar a la base de datos\: 
AddSequenceDialog.UseDatabaseGroup.Label=Utilizar una base de datos para generar la secuencia
AddSequence.Log.TransformationCountersHashtableNotAllocated=Error interno\: no se ha asignado la tabla hash de contadores de transformaci\u00F3n
AddSequenceDialog.UseDatabase.Tooltip=Seleccione si quiere utilizar una base de datos para obtener el valor de la secuencia.
AddSequenceMeta.Exception.UnableToReadStepInfo=No se ha podido leer la informaci\u00F3n del paso desde el repositorio para id_step\=
AddSequenceMeta.CheckResult.NoInputReceived.Title=\u00A1No se recibe ninguna entrada de otros pasos\!
AddSequenceDialog.StartAt.Label=Valor inicial 
AddSequenceDialog.UseCounter.Label=\u00BFUtilizar contador para calcular secuencia? 
AddSequence.Log.NeedToSelectSequence=Debe seleccionar o crear una secuencia utilizando una base de datos o un contador interno.
AddSequenceDialog.Valuename.Label=Nombre del valor 
AddSequenceDialog.NoValidConnectionError.DialogTitle=ERROR
AddSequence.Exception.NoSpecifiedMethod=\u00A1No se ha especificado ning\u00FAn m\u00E9todo en este paso\!
AddSequence.Log.ReadRow=Se ley\u00F3 la fila \#
AddSequenceDialog.UseCounterGroup.Label=Utilizar un contador de la transformaci\u00F3n para generar la secuencia
AddSequenceDialog.CounterName.Label=Nombre del contador (opcional)
AddSequenceDialog.IncrBy.Label=Incrementar en 
AddSequenceMeta.ErrorMessage.NoConnectionDefined=No se ha definido una conexi\u00F3n en este paso.
AddSequenceDialog.Shell.Title=Obtener valor de secuencia de la base de datos
AddSequence.Exception.ErrorReadingSequence=Error leyendo el siguiente valor de la secuencia [{0}] de la base de datos
AddSequenceMeta.CheckResult.SequenceCouldNotBeFound.Title=No se ha podido encontrar la secuencia [{0}]
AddSequenceDialog.Seqname.Label=Nombre de secuencia 
AddSequenceDialog.StepName.Label=Nombre de paso 
AddSequenceMeta.Exception.ErrorLoadingStepInfo=Error cargando paso desde XML
AddSequence.Log.LineNumber=N\u00FAmero de l\u00EDnea 
AddSequenceMeta.Exception.UnableToSaveStepInfo=No se ha podido guardar la informaci\u00F3n del paso al repositorio para id_step\=
AddSequenceDialog.MaxVal.Label=Valor m\u00E1ximo 
AddSequenceMeta.CheckResult.SequenceExists.Title=La secuencia ya existe.
AddSequenceMeta.CheckResult.StepIsReceving.Title=Este paso recibe informaci\u00F3n de otros pasos.
AddSequenceMeta.CheckResult.UnableToConnectDB.Title=No se ha podido conectar a la base de datos para verificar la secuencia debido a un error\: 
